sources/Common/CMakeLists.txt

```diff
@@ -28,8 +28,6 @@ check_env_var(DYNAFLOW_LAUNCHER_LOCALE)
 # We use a target to ensure it will be executed at each build
 add_custom_target(dicokeys
 BYPRODUCTS ${CMAKE_CURRENT_SOURCE_DIR}/src/DicoKeys.cpp ${CMAKE_CURRENT_SOURCE_DIR}/include/DicoKeys.h
-COMMAND rm -f ${CMAKE_CURRENT_SOURCE_DIR}/src/DicoKeys.cpp
```

Comment:
Useless as the file is systematically erase by the script right?

Reply:
Yes, it is always recreated even if it already exists